src/energyplus/ForwardTranslator/ForwardTranslateThermalZone.cpp

```diff
@@ -889,14 +889,15 @@ namespace energyplus {
             double outdoorAirFlowAirChangesperHour = designSpecificationOutdoorAir->outdoorAirFlowAirChangesperHour();
 
             std::string outdoorAirMethod = designSpecificationOutdoorAir->outdoorAirMethod();
-            if (istringEqual(outdoorAirMethod, "Max")) {
+            if (istringEqual(outdoorAirMethod, "Maximum")) {
```

Comment:
Issue 2 on #4461: this was NEVER true since "Maximum" is one of the valid methods, not "Max"